client/go/Taskfile.yml

```diff
@@ -65,6 +65,10 @@ tasks:
     desc: "Build the tun2socks binary and library for Linux"
     cmds: [{task: electron, vars: {TARGET_OS: "linux", TARGET_ARCH: "amd64"}}]
 
+  linux_arm64:
```

Comment:
We need to update "line 49" and "line 55" to make sure we are using the `aarch64` architecture of `zig`. Here is the example `CC`:

```
CC='zig cc -target aarch64-linux-gnu'
```